src/openrct2/config/IniWriter.hpp

```diff
@@ -34,7 +34,7 @@ interface IIniWriter
         std::string key = configEnum.GetName(value);
         if (key.empty())
         {
-            WriteInt32(name, value);
+            WriteInt32(name, static_cast<int32_t>(value));
```

Comment:
This is the first time we've had a config enum that wasn't backed by `int32_t`.  I *think* that, after we've made all of the config enums strong, this code path becomes unhittable...

Reply:
We could add a static_assert for when the underlying type is bigger.

Reply:
Added in 788bc15.